### PR TITLE
Add xdebug & debugger config to stack

### LINKED ISSUE #16: Add xedbug & debug
In the latest projects (FWT, Vidy, ...) I included xdebug in the stack which help me quite a lot to debug and dev on the php side more effectively.

Do you see any problem to include it back in the boilerplate?

If not I'll create a PR

### PR DESCRIPTION
**Pull request by kuuak**
Fixes #16 

I also included a custom pre-start script to download a local version of the WordPress file for the PHP debugger to work properly.
This script will automatically download the same version as the version in the Dockerfile

⚠️ PR #30 needs to be merged before this one

### CODE REVIEW

File: wordpress/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
+    "prestart": "sh ./scripts/pre-start.sh",
```
Comment (charl0tee) [inline wordpress/package.json:11]: @kuuak Is it a script that we have to launch ourself ? I think we should also add some documentation about this feature (+ this script if needed to run manually)

**Review by charl0tee — CHANGES_REQUESTED** [review]

File: wordpress/package.json (hunk 1 above)
Comment (kuuak) [reply wordpress/package.json:11]: @charl0tee The `prestart` script is run automatially every time `start` script is executed (just before as it's name suggests). See npm doc:

- [life-cycle-scripts](https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts)
- [npm-start](https://docs.npmjs.com/cli/v6/using-npm/scripts#npm-start)

Comment (charl0tee) [reply wordpress/package.json:11]: @kuuak Ah ok ! Niice ! :👌

**Review by charl0tee — DISMISSED** [review]
All good to me ! Just wondering if we still need to add some documentation about debugging or if it's not relevant as it's not really related to our stack

**Comment by kuuak** [comment]
@charl0tee @snugglejuice I added some doc ;) https://github.com/superhuit-agency/superstack/pull/35/commits/5bea56af2659a1abad8316b409607729c28a433f

**Review by charl0tee — APPROVED** [review]
👏🏽

**Pull request merged**